core/common/buffer.hpp

```diff
@@ -189,6 +189,16 @@ namespace kagome::common {
 
     std::vector<uint8_t> &toVector();
 
+    /**
+     * @brief getter for span of const bytes
+     */
+    gsl::span<const uint8_t> toConstSpan() const;
+
+    /**
+     * @brief getter for span of mutable bytes
+     */
+    gsl::span<uint8_t> toSpan();
+
```

Comment:
Why we need these methods? Isn't buffer itself convertible to span? if not can we make it convertible?

Reply:
It is convertible, but when we use buffer with mocked functions we have to provide the exact type, it doesn't convert implicitly.